sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py

```diff
@@ -546,85 +546,74 @@ def request_id(self):
+    :paramtype lifetime_actions: Iterable[~azure.keyvault.certificates.LifetimeAction]
+    :keyword str certificate_type: Type of certificate to be requested from the issuer provider.
+    :keyword bool certificate_transparency: Indicates if the certificates generated under this policy
         should be published to certificate transparency logs.
-    :keyword Iterable[str] san_emails: Subject alternative emails of the X509 object. Only one out
-        of san_emails, san_dns_names, and san_upns may be set.
-    :keyword Iterable[str] san_dns_names: Subject alternative DNS names of the X509 object. Only one out
-        of san_emails, san_dns_names, and san_upns may be set.
-    :keyword Iterable[str] san_upns: Subject alternative user principal names of the X509 object. Only one out
-        of san_emails, san_dns_names, and san_upns may be set.
+
     """
 
     # pylint:disable=too-many-instance-attributes
     def __init__(
         self,
         issuer_name,  # type: str
-        subject_name,  # type: str
-        exportable=None,  # type: Optional[bool]
-        key_type=None,  # type: Optional[KeyType]
-        key_size=None,  # type: Optional[str]
-        reuse_key=None,  # type: Optional[bool]
-        curve=None,  # type: Optional[KeyCurveName]
-        ekus=None,  # type: Optional[list[str]]
-        key_usage=None,  # type: Optional[list[Union[KeyUsageType, str]]]
-        content_type=None,  # type: Optional[Union[SecretContentType, str]]
-        validity_in_months=None,  # type: Optional[int]
-        lifetime_actions=None,  # type: Optional[list[LifetimeAction]]
-        certificate_type=None,  # type: Optional[str]
-        certificate_transparency=None,  # type: Optional[bool]
         **kwargs  # type: **Any
     ):
         # type: (...) -> None
-        self._subject_name = subject_name
-        self._attributes = kwargs.get("attributes", None)
-        self._id = kwargs.get("cert_policy_id", None)
-        self._exportable = exportable
-        self._key_type = key_type
-        self._key_size = key_size
-        self._reuse_key = reuse_key
-        self._curve = curve
-        self._ekus = ekus
-        self._key_usage = key_usage
-        self._content_type = content_type
-        self._validity_in_months = validity_in_months
-        self._lifetime_actions = lifetime_actions
         self._issuer_name = issuer_name
-        self._certificate_type = certificate_type
-        self._certificate_transparency = certificate_transparency
-        self._san_emails = kwargs.pop("san_emails", None)
-        self._san_dns_names = kwargs.pop("san_dns_names", None)
-        self._san_upns = kwargs.pop("san_upns", None)
-
-        sans = [self._san_emails, self._san_upns, self._san_dns_names]
-        if len([x for x in sans if x is not None]) > 1:
-            raise ValueError("You can only set at most one of san_emails, san_dns_names, and san_upns")
+        self._subject_name = kwargs.pop("subject_name", None)
```

Comment:
Removing a required positional parameter is a breaking change. However, it appears this didn't break a test.

Reply:
added tests for this, thanks!